pyiceberg/cli/console.py

```diff
@@ -64,10 +72,16 @@ def run(
     catalog: str | None,
     verbose: bool,
     output: str,
+    log_level: str,
     ugi: str | None,
     uri: str | None,
     credential: str | None,
 ) -> None:
+    logging.basicConfig(
+        level=getattr(logging, log_level.upper()),
+        format="%(levelname)s:%(name)s:%(message)s",
```

Comment:
Could you add a timestamp, line no to this?
`format="%(asctime)s:%(levelname)s:%(name)s:%(lineno)d:%(message)s"` something like this?
Or better yet structured logging - not sure if that requires a larger change?

Reply:
thanks! i added the timestamp. I think line number might be too verbose for the CLI usecase